cvmfs/util/posix.cc

```diff
@@ -1573,6 +1573,48 @@ bool SafeWriteToFile(const std::string &content,
   return retval;
 }
 
+void Panic() {
+#ifdef LIB_CVMFS_SERVER
+  throw "panic";
```

Comment:
We should have an exception base class that derives from `std::runtime_error`.  Perhaps `ECvmfsException`